authors/apps/authentication/views.py

```diff
@@ -17,7 +20,7 @@ class RegistrationAPIView(APIView):
     serializer_class = RegistrationSerializer
 
     def post(self, request):
-        user = request.data.get('user', {})
+        user = request.data
 
         # The create serializer, validate serializer, save serializer pattern
```

Comment:
@BarnaTB Update the comments here to what the method actually does and place this on top of the method.